doc/changes/devel/12336.bugfix.rst

```diff
@@ -0,0 +1 @@
+Allow :meth:`mne.io.Raw.interpolate_bads` and :meth:`mne.Epochs.interpolate_bads` to work on ``ecog`` and ``seeg`` data, by `Alex Rockhill`_
```

Comment:
I feel you need to explain how interpolation works. For EEG spline refers to spherical splines after mapping the EEG sensors to a spherical coordinate system.